content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,46 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+<!-- keywords: data science project architecture, machine learning project architecture, machine learning workflow, data science workflow, machine learning file system, data science file system, data science project structure, machine learning project structure, notebook version control -->
+
+# Workspace
+
+A data science project consists of data obtained from many different sources.
+Most of the time it needs to convert the the format of this data into a form
+that is required by the training models and supplying to data science / machine
+learning workflows. Sometimes it needs to be split into multiple files or
+directories or (as the project structure needs) have different versions for
+different requirements. e.g. A smaller / simplified version might be required in
+protoyping for faster feedback and shorter training times.
+
+These activities all involve to change the machine learning / data science
+project structure in unforeseeable ways and tracking them manually becomes
+tedious. A single workspace to manage all artifacts of a project is desirable,
+although versioning needs and managing dependencies make it increasingly
+complex.
+
+DVC allows a single directory to contain all your project artifacts. In the
+documentation the workspace is the _user visible_ part of the directory that
+contains all your project files e.g. raw datasets, source code, ML models, etc.
+Users work in this directory using their data and model files that and
```

Comment:
> the **workspace** is the _user visible_ part of the **DVC project**

I would try to get to this rather quickly (after a short intro p if needed). This is the main content.